features/steps/util.py

```diff
@@ -85,6 +88,30 @@ def parse_csv(text: str) -> list[str]:
     return re.findall(r"\"(.+?)\"\s*,?", text)
 
 
+def execute_notebook(notebook_filename):
```

Comment:
Is this used anywhere? Or is this left-over from trying to get the notebook working in the e2e tests?

Reply:
removed